realsense2_camera/include/realsense2_camera/realsense_node.h

```diff
@@ -151,6 +152,7 @@ class RealSenseParamManager;
         static std::string getNamespaceStr();
         void getParameters();
         bool enableStreams(std_srvs::SetBool::Request  &req, std_srvs::SetBool::Response &res);
+        bool JsonConfigCallback(JsonConfig::RequestType &request, realsense2_camera::JsonConfig::ResponseType &response);
```

Comment:
Inconsistent use of the `realsense2_camera` namespace

Reply:
Nice catch will fix it!